packages/grafana-ui/package.json

```diff
@@ -55,6 +55,7 @@
     "hoist-non-react-statics": "3.3.2",
     "immutable": "3.8.2",
     "jquery": "3.5.1",
+    "@kusto/monaco-kusto": "3.2.7",
```

Comment:
isn't this the one that weights a ton? ;)

Reply:
Yup 😇 

It's lazy loaded though just when using kusto syntax :)